packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -240,15 +262,26 @@ class ScrollableBase extends Component {
 		if (pos) {
 			const
 				{top, left} = pos,
-				bounds = this.uiRef.getScrollBounds();
+				bounds = this.uiRef.getScrollBounds(),
+				horizontalMove = bounds.maxLeft > 0 && left !== this.uiRef.scrollLeft,
+				verticalMove = bounds.maxTop > 0 && top !== this.uiRef.scrollTop;
```

Comment:
How about `canMoveHorizontally`, `canMoveVertically` for `horizontalMove `, `verticalMove ` like `canScrollVertically`?

Reply:
Good!

Reply:
I will change them to `scrollHorizontally' and 'scrollVertically' since they are not for scrollability but for actual scrolling.